buildh/UI.py

```diff
@@ -18,6 +18,8 @@
 from . import writers
 from . import utils
 
+__version__ = "1.3.1"
+
```

Comment:
`from . import __version__` should to the trick

Reply:
Thanks @HubLot 
It works!